src/config/user.cr

```diff
@@ -16,7 +16,7 @@ struct DppmRestApi::Config::User
   end
 
   def self.new(api_key_hash string : String,
-               groups : Set(Int32),
+               groups : Enumerable(Int32),
```

Comment:
Why changing`Set` to `Enumerable`?

Reply:
Because `#to_set` is called on group before passing it to the next constructor, so this constructor can take *any* enumerable type and it will be fine.